client/components/LearnerViewDirective.js

```diff
@@ -517,9 +534,29 @@ tie.directive('learnerView', [function() {
           loadQuestion(questionId, questionSet.getIntroductionParagraphs());
         };
 
+        var displayAutoSaveTextForSeconds = function(displaySeconds) {
+          $scope.displayAutoSaveText = true;
+          $scope.$apply();
```

Comment:
Somehow, if I don't do $scope.$apply, displayAutoSaveText won't be changed. I will double check, but I am pretty sure it wasn't working without $scope.$apply().